File: bootstrap/v0.0.8.4/benchmarks/java/Bubblesort.java
public class Bubblesort {
    public static void main(String[] args) {
        int n = 1000;
        int[] arr = new int[n];

        for (int i = 0; i < n; i++) {
            arr[i] = n - i;
        }

        boolean sorted = false;
        while (!sorted) {
            sorted = true;
            for (int i = 0; i < n - 1; i++) {
                if (arr[i] > arr[i + 1]) {
                    int temp = arr[i];
                    arr[i] = arr[i + 1];
                    arr[i + 1] = temp;
                    sorted = false;
                }
            }
        }

        System.exit(0);
    }
}

File: bootstrap/v0.0.8.5/benchmarks/rust/bubblesort.rs
fn main() {
    let n = 1000;
    let mut arr: Vec<i32> = (0..n).map(|i| n - i).collect();

    let mut sorted = false;
    while !sorted {
        sorted = true;
        for i in 0..n - 1 {
            if arr[i as usize] > arr[(i + 1) as usize] {
                arr.swap(i as usize, (i + 1) as usize);
                sorted = false;
            }
        }
    }

    std::process::exit(0);
}
